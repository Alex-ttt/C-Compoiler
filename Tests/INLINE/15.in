void main() {
	int x = 5;
	{
		int y = x;
		printf("%d", y);
	}
}
