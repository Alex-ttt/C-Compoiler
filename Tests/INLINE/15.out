 Symbols Table:
   main  void function main(
  )
    {
      Statements:
        x
      =
        5
--------------------------
      {
        Statements:
          y
        =
          x
--------------------------
        printf(
          "%d"
        ,
            y
        )
--------------------------
        Symbols Table:
                  y int
      }
--------------------------
      Symbols Table:
              x int
    }
